docs/Operators-ml.md

```diff
@@ -537,38 +545,38 @@ This version of the operator has been available since version 1 of the 'ai.onnx.
 
 <dl>
 <dt><tt>cats_int64s</tt> : list of ints</dt>
-<dd>list of categories, ints</dd>
+<dd>List of categories, ints.<br>One and only one of the 'cats_*' attributes must be defined.</dd>
 <dt><tt>cats_strings</tt> : list of strings</dt>
-<dd>list of categories, strings</dd>
+<dd>List of categories, strings.<br>One and only one of the 'cats_*' attributes must be defined.</dd>
 <dt><tt>zeros</tt> : int</dt>
-<dd>if true and category is not present, will return all zeros, if false and missing category, operator will return false. Default is true (1).</dd>
+<dd>If true and category is not present, will return all zeros; if false and a category if not found, the operator will fail.</dd>
 </dl>
 
 #### Inputs
 
 <dl>
 <dt><tt>X</tt> : T</dt>
-<dd>Data to be encoded</dd>
+<dd>Data to be encoded.</dd>
 </dl>
 
 #### Outputs
 
 <dl>
 <dt><tt>Y</tt> : tensor(float)</dt>
-<dd>encoded output data</dd>
+<dd>Encoded output data, having one more dimension than X.</dd>
 </dl>
 
 #### Type Constraints
 
 <dl>
 <dt><tt>T</tt> : tensor(string), tensor(int64), tensor(int32), tensor(float), tensor(double)</dt>
-<dd> allowed types.</dd>
+<dd>The input must be a tensor of a numeric type.</dd>
 </dl>
 
 
 ### <a name="ai.onnx.ml.SVMClassifier"></a><a name="ai.onnx.ml.svmclassifier">**ai.onnx.ml.SVMClassifier**</a>
 
-  SVM classifier prediction
+  Support Vector Machine classifier
```

Comment:
What are its input and output restrictions? I feel like we need to have a input/output format for all traditional classifiers and regressor somewhere. The format document may include something like:
- Input shape can be [C] or [N, C]
- For classifiers, if the input shape is [C], the output label/probabilities shape will be [1]/[L]. If the input shape becomes [N, C] the output labels/probabilities shape will be [N]/[N, L], where L is the number of total labels.
- For regressors with a scalar output, if the input shape is [C], the output shape will be [1]. If the input shape becomes [N, C] the output shape will be [N].
- For regressors with a vector output, if the input shape is [C], the output shape will be [L]. If the input shape becomes [N, C] the output shape will be [N, L], where L is the length of an example's output vector.

Reply:
I believe it'd be better to repeat it for each operator because the definitions of different operators are essentially independent. A shortcut is to define on an operator (for example, only specify those restrictions for linear classifier and regressor) and the refer it at other places.